### PR TITLE
[4] Remove deprecated function calls and use short array syntax

### PR DESCRIPTION
**Pull request by PhilETaylor**
### Summary of Changes

Remove deprecated function calls and use short array syntax

Instead of `Factory::getUser()` which is deprecated use `Factory::getApplication()->getIdentity()`

Use short array syntax `[]` instead of long syntax `array()`

### Testing Instructions

Code Review only

@Bakual  :)

### CODE REVIEW

File: administrator/components/com_messages/tmpl/messages/blankstate.php [hunk 1]
```diff
@@ -12,13 +12,13 @@
 use Joomla\CMS\Factory;
 use Joomla\CMS\Layout\LayoutHelper;
 
-$displayData = array(
-		'textPrefix' => 'COM_MESSAGES',
-		'formURL'    => 'index.php?option=com_messages&view=messages',
-		'helpURL'    => 'https://docs.joomla.org/Special:MyLanguage/Help40:Private_Messages',
-);
+$displayData =[
```
Comment (Quy) [inline administrator/components/com_messages/tmpl/messages/blankstate.php]:
```suggestion
$displayData = [
```

**Review by Bakual — APPROVED** [review]
Sure thing, fine for me!

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on edabb1abcd9ffc8333a4d2d6b04db1c73753435a<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33322">issues.joomla.org/tracker/joomla-cms/33322</a>.</sub>

**Comment by joomdonation** [comment]
I have tested this item :white_check_mark: successfully on edabb1abcd9ffc8333a4d2d6b04db1c73753435a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33322">issues.joomla.org/tracker/joomla-cms/33322</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33322">issues.joomla.org/tracker/joomla-cms/33322</a>.</sub>

**Pull request merged**